app/templates/partials/answers/unit.html

```diff
@@ -22,6 +22,6 @@
         'id': answer.id
         } %}
         {% include 'partials/forms/input.html' %}
-        <span class="input-type__type" id="{{answer.id}}-type">{{ format_unit_input_label(answer.unit, unit_length=answer.unit_length) }}</span>
+        <abbr title="{{ _(label.text) }}" class="input-type__type" id="{{answer.id}}-type">{{ format_unit_input_label(answer.unit, unit_length=answer.unit_length) }}</abbr>
```

Comment:
It would be clearer if `label.text` was `answer.label`. The label variable is constructed to pass to the label template.